charm-slurmctld/src/interface_slurmdbd.py

```diff
@@ -79,28 +79,30 @@ def _on_relation_changed(self, event):
         if event_app_data:
             slurmdbd_info = event_app_data.get("slurmdbd_info")
             if slurmdbd_info:
-                self._charm.set_slurmdbd_available(True)
                 self.on.slurmdbd_available.emit()
             else:
                 event.defer()
         else:
             event.defer()
 
     def _on_relation_departed(self, event):
-        self._charm.set_slurmdbd_available(False)
         self.on.slurmdbd_unavailable.emit()
 
     def _on_relation_broken(self, event):
         if self.framework.model.unit.is_leader():
             event.relation.data[self.model.app]["munge_key"] = ""
             event.relation.data[self.model.app]["jwt_rsa"] = ""
-        self._charm.set_slurmdbd_available(False)
         self.on.slurmdbd_unavailable.emit()
 
     @property
     def _relation(self):
         return self.framework.model.get_relation(self._relation_name)
 
+    @property
+    def is_joined(self):
+        """Return True if self._relation is not None."""
+        return len(self._charm.framework.model.relations[self._relation_name]) > 0
```

Comment:
How do you feel about
```suggestion
        return True is self._charm.framework.model.relations.get(self._relation_name) else False
```
?

Reply:
I'm not a big fan of inline conditionals. Do you prefer it over a "conventional" multiline `if`?